aries-site/src/examples/components/rangeinput/RangeInputInFormExample.js

```diff
@@ -0,0 +1,139 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  CheckBoxGroup,
+  DateInput,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Text,
+  TextInput,
+  ResponsiveContext,
+  Select,
+  RangeInput,
+} from 'grommet';
+
+export const RangeInputInFormExample = () => {
+  const [guests, setGuests] = React.useState();
+  const [price, setPrice] = React.useState('200');
+  const [dates, setDates] = React.useState([
+    '2020-07-31T15:27:42.920Z',
+    '2020-08-07T15:27:42.920Z',
+  ]);
+  const size = useContext(ResponsiveContext);
+
+  const onChange = event => {
+    const nextDates = event.value;
+    setDates(nextDates);
+  };
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <Box gap="medium" width="medium">
+      <Header
+        direction="column"
+        align="start"
+        gap="xxsmall"
+        pad={{ horizontal: 'xxsmall' }}
+      >
+        {/* Use semantically correct heading level and adjust size as 
+        needed. In this instance, this example is presented within an 
+        HTML section element and this is the first heading within the 
+        section, therefor h2 is the semantically correct heading. For 
+        additional detail, see https://design-system.hpe.design/foundation/typography#semantic-usage-of-heading-levels). */}
+        <Heading level={2} margin="none">
+          Book a Hotel
```

Comment:
We should come up with a more "enterprise focused" / HPE relevant example. A recurring critique / ask of the Design System is to present more relevant examples to HPE use cases.

Reply:
Sounds good Ill chat with @vavalos5  about this so we can line up figma with another example

Reply:
Agreed @halocline i'll peek around other files to find where we might be using `RangeInput` within a form